Tests/ResilientDecodingTests/BugTests.swift

```diff
@@ -0,0 +1,52 @@
+// Created by George Leontiev on 5/6/20.
+// Copyright © 2020 Airbnb Inc. All rights reserved.
+
+import XCTest
+import ResilientDecoding
+
+/**
+ Tests for bugs that were encountered after releasing. The commit which introduces a test here should also introduce a fix.
+ */
+final class BugTests: XCTestCase {
+  func testURLs() throws {
```

Comment:
can we make this title more explicit? I'm not sure what exactly we're testing here. Maybe we could include a description of the prior bug in a comment above this method?